### PR TITLE
Feature added in slcli block snapshot-cancel, Text updated in slcli block snapshot-enable

### PR DESCRIPTION
**Pull request by ramkishor-ch**
Hi @allmightyspiff,

Fixes: #2013 

**Title:**
Text update and option needs to added in slcli block snapshot-cancel, ./slcli block snapshot-enable

**Description:**
1. I have added feature called force in ./slcli block snapshot-cancel --help 
2. I have updated the text for --minute, --hour, --day-of-week in ./slcli block snapshot-enable --help.

Please find the screen shot.

Please review it.

**Screen Shot:**
Screenshot 2023-08-02 at 4 16 16 PM

Thanks,
Ramkishor Chaladi.

### CODE REVIEW

File: SoftLayer/CLI/block/snapshot/enable.py [hunk 1]
```diff
@@ -16,13 +16,16 @@
               help='Number of snapshots to retain.',
               required=True)
 @click.option('--minute',
-              help='Minute of the day when snapshots should be taken.',
+              help='Minute of the hour when snapshots should be taken, '
+              'integer between 0 to 59.',
```
Comment (allmightyspiff) [inline SoftLayer/CLI/block/snapshot/enable.py:20]:
```suggestion
              help='Minute of the hour when snapshots should be taken, integer between 0 to 59.',
```
for this project the line length is 120 characters, so I prefer to keep things on one line if possible.

File: SoftLayer/CLI/block/snapshot/enable.py [hunk 2]
```diff
@@ -16,13 +16,16 @@
...
               default=0)
 @click.option('--hour',
-              help='Hour of the day when snapshots should be taken.',
+              help='Hour of the day when snapshots should be taken, '
+              'integer between 0 to 23.',
```
Comment (allmightyspiff) [inline SoftLayer/CLI/block/snapshot/enable.py:24]:
```suggestion
              help='Hour of the day when snapshots should be taken, integer between 0 to 23.',
```

File: SoftLayer/CLI/block/snapshot/enable.py [hunk 3]
```diff
@@ -16,13 +16,16 @@
...
               default=0)
 @click.option('--day-of-week',
-              help='Day of the week when snapshots should be taken.',
+              help='Day of the week when snapshots should be taken, '
+              'integer between 0 to 6',
```
Comment (allmightyspiff) [inline SoftLayer/CLI/block/snapshot/enable.py:28]:
```suggestion
              help='Day of the week when snapshots should be taken, integer between 0 to 6',
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, just make the help message one line since the length is <120 characters.

**Comment by ramkishor-ch** [comment]
Hi @allmightyspiff,

As suggested, I have updated and re-tested it.

Please find the screen shot.

Screen Shot:
Screenshot 2023-08-04 at 4 27 37 PM

Please re-review it.

Thanks,
Ramkishor Chaladi.

**Pull request merged**